nixos/modules/services/web-servers/minio.nix

```diff
@@ -69,6 +70,17 @@ in
       type = types.package;
       description = "Minio package to use.";
     };
+
+    extraConfig = mkOption {
+        type = types.attrsOf types.str;
```

Comment:
Indentation is off. 2 spaces per level please.

Reply:
No longer relevant